platform/platform_windows.go

```diff
@@ -95,6 +96,62 @@ func netServerGetInfo() (si SERVER_INFO_101, err error) {
 	return platGetServerInfo(outdata), nil
 }
 
+func fetchOsDescription() (string, error) {
```

Comment:
Maybe a good idea to link here to the SO question where you found the first method we try?